src/vivarium_public_health/risks/data_transformations.py

```diff
@@ -486,6 +471,9 @@ def validate_relative_risk_rebin_source(
 
 def validate_rebin_source(builder, risk: EntityString, data: pd.DataFrame):
 
+    if not isinstance(data, pd.DataFrame):
+        return
```

Comment:
nit: maybe it's just a me thing but I dislike not saying the type of return (even if it's the implied None). We don't do it elsewhere so maybe worth doing just for consistency